docs/advanced.md

```diff
@@ -1105,27 +1106,28 @@ global variable will indicate the completion status of the command.
 
 If the command to be executed is not found, `Errno::ENOENT` is raised.
 
-### Using the sh action
+### Using the sh method
 
-You can also use the built-in `sh` Fastlane action:
+You can also use the built-in `Actions.sh` method:
```

Comment:
Why call is "`Actions.sh` method" if I use it by just calling `sh`?

Reply:
For one thing, this introduces the method and so provides more information about where it comes from. (It took me a while to track down where all this is happening.) It happens to be available in any class method in the `Actions` namespace, so within an action (including in a plugin) it can just be called as `sh`. If you're in a helper, e.g., you cannot just call `sh`. It has to be `Actions.sh` (or even `Fastlane::Actions.sh`). I think it's valuable to mention `Actions.sh` once.

Reply:
Makes me think of a shell script named `Actions` that has the standard `sh` extension.